lib/src/github_sign_in_page.dart

```diff
@@ -22,53 +24,47 @@ class GitHubSignInPage extends StatefulWidget {
 }
 
 class _GitHubSignInPageState extends State<GitHubSignInPage> {
-  final FlutterWebviewPlugin _wv = FlutterWebviewPlugin();
-
   static const String _userAgentMacOSX =
       "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.88 Safari/537.36";
 
   @override
   void initState() {
     super.initState();
-
-    _wv.onUrlChanged.listen((url) {
-      if (url.startsWith(widget.redirectUrl)) {
-        Navigator.of(context)
-            .pop(url.replaceFirst("${widget.redirectUrl}?code=", "").trim());
-      } else if (url.contains("error=")) {
-        Navigator.of(context).pop(
-          Exception(Uri.parse(url).queryParameters["error"]),
-        );
-      }
-    });
+    if (kIsWeb) {
```

Comment:
In case kIsWeb is false, what will happen? Do you have any plan for fallback?

Reply:
Woops I did not mean to actually merge this !

Reply:
I was just testing it. Let me fix it and get back !

Reply:
> In case kIsWeb is false, what will happen? Do you have any plan for fallback?

If this is false, this means it is the other platforms (mobile or desktop), in which it will default to the normal code that is already there